tests/interfaces/python/test_5_tcopflow.py

```diff
@@ -0,0 +1,38 @@
+import exago
```

Comment:
Move this import to after `check_preconditions()` is called.

Also, make sure to extend this fix to all of our other tests

Reply:
Please make sure to move this import.........